File: register-app/src/main/java/com/realize/register_app/MainActivity.java
package com.realize.register_app;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

import com.realize.routeeasy.utils.Router;
import com.realize.router_note.EAction;

@EAction(path = "register/action")
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        TextView tvButton = findViewById(R.id.tv_button);
        tvButton.setOnClickListener(v ->
                Router.getInstance().startAction("fragment/test?asb=iushdksd"));
    }
}

File: login-app/src/main/java/com/realize/login_app/MainActivity.kt
package com.realize.login_app

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.realize.router_note.EAction

@EAction(path = "login/main")
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}